app/views/identified_clients/_assessments.haml

```diff
@@ -0,0 +1,7 @@
+- if @non_hmis_client.non_hmis_assessments.count > 1
```

Comment:
If we set `@assessments = @non_hmis_client.non_hmis_assessments`, `@client = @non_hmis_client`, and `@path = identified_client_path` in the controller could we streamline the views and just have one instance of most of them?